test/ex_integrate_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule ExIntegrateTest do
+  use ExUnit.Case
+  doctest ExIntegrate
+
+  test "run steps" do
+    assert ExIntegrate.run_steps("test/ei.test.json") == :ok
```

Comment:
I think the convention I'm used to seeing for checking the output of an impure function is: 
```elixir
assert :ok = ExIntegrate.run_steps("test/ei.test.json")
```

Reply:
Probably doesn't make much of a difference 🤷‍♂️ .